web/src/routes/PoisPage.tsx

```diff
@@ -106,12 +115,13 @@ const PoisPage = ({ match, cityModel, location, languages, history }: PropsType)
   const sortedPois = pois.sort((poi1: PoiModel, poi2: PoiModel) => poi1.title.localeCompare(poi2.title))
   const renderPoiListItem = (poi: PoiModel) => <PoiListItem key={poi.path} poi={poi} />
   const pageTitle = `${t('pageTitle')} - ${cityModel.name}`
+  const featureLocations = pois.map(poi => poi.featureLocation).filter(poi => poi) as Feature[]
```

Comment:
because does not accept null
```js
embedInCollection = (features: Feature[])
```
i could also disable the null check for the feature collection or adjust the embedFkt
```js
 const featureLocations = pois.map(poi => poi.featureLocation!).filter(poi => poi)
```

Reply:
you can say: 
`.filter((feature): feature is Feature => !!feature)`